cpp/src/io/parquet/page_enc.cu

```diff
@@ -534,6 +541,9 @@ CUDF_KERNEL void __launch_bounds__(128)
   uint32_t const t          = threadIdx.x;
   auto const data_page_type = write_v2_headers ? PageType::DATA_PAGE_V2 : PageType::DATA_PAGE;
 
+  // Max page header size excluding statistics
+  auto const max_data_page_hdr_size = write_v2_headers ? MAX_V2_HDR_SIZE : MAX_V1_HDR_SIZE;
```

Comment:
this is unrelated to the WAR, right?

Reply:
Yes, it's replacing (incorrect as it turns out) magic numbers I noticed while tracking down the bug.